libraries/cli/include/cli/testnet_config.hpp

```diff
@@ -12,29 +12,41 @@ const char *testnet_json = R"foo({
   "network_simulated_delay": 0,
   "network_transaction_interval": 100,
   "network_bandwidth": 40,
-  "network_ideal_peer_count": 5,
-  "network_max_peer_count": 15,
-  "network_sync_level_size": 25,
+  "network_ideal_peer_count": 10,
+  "network_max_peer_count": 50,
+  "network_sync_level_size": 10,
   "network_packets_processing_threads": 14,
   "network_peer_blacklist_timeout" : 600,
   "deep_syncing_threshold" : 10,
   "network_boot_nodes": [
     {
       "id": "f36f467529fe91a750dfdc8086fd0d2f30bad9f55a5800b6b4aa603c7787501db78dc4ac1bf3cf16e42af7c2ebb53648653013c3da1987494960d751871d598a",
       "ip": "boot-node-0.testnet.taraxa.io",
-      "tcp_port": 10002,
       "udp_port": 10002
     },
     {
       "id": "d2d445fc3276bdbf2a07628a484baabf45ccb91d6aa078a0dc3aefc11f1941899a923312ec0e664b8e57b63b774c47a006d9d1d16befd2135dcf76067736c688",
       "ip": "boot-node-1.testnet.taraxa.io",
-      "tcp_port": 10002,
       "udp_port": 10002
     },
     {
       "id": "c6e7263f44d88c0d6cc3b0d5ebdc31cf891908e8fa7e545e137d3ed0bfec1810fa24c1379228afbb53df0d59e716e17138115fd096782a84261718ab77665171",
       "ip": "boot-node-2.testnet.taraxa.io",
-      "tcp_port": 10002,
+      "udp_port": 10002
+    },
+    {
+      "id": "dcc7f60f64eee9fc470e24fd821b3d82acca646c6c951169a8f5b3de297e241cd9987e1cca1083c85b6482ba09715bbea8765710b149c55493e0bb16fc1c29cc",
+      "ip": "taraxa-node-0.testnet.taraxa.io",
+      "udp_port": 10002
+    },
+    {
+      "id": "dedae37d9c96e0f2c8ef2796351f5560234a3e8e0407ef5cd16aaf93dd8ffd437608a585d5dc3eea54e30e1392591cfc61c6fce3cbf12b46dcc479b5fbdcde96",
+      "ip": "taraxa-node-1.testnet.taraxa.io",
```

Comment:
we have duplicate `taraxa-node-<N>.testnet.taraxa.io` in this config with different `id's`

Reply:
added lines is related to `taraxa-node-*`, but old ones is `boot-node-*`. Or am I missing something?

Reply:
It definitely shouldn't make anything worse. I think we should discuss it with @rattrap

Reply:
sure. In my understanding adding taraxa nodes was just "quick fix" that we created just to make it somehow work. It should not be used at all after we fixed the bootnodes